mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot_test.go

```diff
@@ -166,3 +170,25 @@ func testAccMongoDBAtlasCloudProviderSnapshotConfig(projectID, clusterName, desc
 		}
 	`, projectID, clusterName, description, retentionInDays)
 }
+
+func TestResourceMongoDBAtlasCloudProviderSnapshot_snapshotID(t *testing.T) {
```

Comment:
Just a note: If this works well then we'll probably want to add this to all import statements since if the project name has hyphens it would fail with any import that isn't ignoring in value hyphens.

Reply:
yes, it would be great to add it to the remaining resources. But I think by the next release because I wouldn't like to break something. 😅